tests/libtest/correct.stdout

```diff
@@ -0,0 +1,84 @@
+{"type":"suite","event":"started","test_count":41}
+{"type":"test","event":"started","name":"Parsing /Users/work/Work/cucumber/tests/features/wait/invalid.feature"}
```

Comment:
@ilslv it feels that it lacks a keyword prepended for each line, like default writer does.

`Feature: Parsing /Users/work/Work/cucumber/tests/features/wait/invalid.feature`

And for Scenario/Background make all this look similar to default output.